devel/addon/kyverno/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+bases:
+  - https://github.com/kyverno/policies/pod-security
+
+# Patch the pod security policy to exclude our E2E Bind deployment which is too
+# difficult to run as non-root
+# TODO(wallrj): This doesn't work because kustomize doesn't properly to
+# strategigMergePatch on custom resources.
+# See https://github.com/kubernetes-sigs/kustomize/issues/1510
+# And https://github.com/kubernetes-sigs/kustomize/issues/2339
+# patches:
+# - target:
+#     kind: ClusterPolicy
+#     name: require-run-as-non-root
+#   patch: |-
+#     apiVersion: kyverno.io/v1
+#     kind: ClusterPolicy
+#     metadata:
+#       name: require-run-as-non-root
+#     spec:
+#       rules:
+#       - name: check-containers
+#         exclude:
+#           resources:
+#             namespaces: ["bind"]
```

Comment:
This was my first attempt to configure Kyverno to ignore certain E2E components, but Kustomize doesn't properly patch CRD resources. 
See https://github.com/kubernetes-sigs/kustomize/issues/1510
and https://github.com/kubernetes-sigs/kustomize/issues/2339

Reply:
I've removed these ineffective patches, to avoid confusion.